index.html

```diff
@@ -88,9 +79,10 @@ <h1 class="fs-3 text-center">Choose Your Category</h1>
           <div class="align-i-ctr flex flex-wrap gap-2p5 justify-c-sa p-4">
             <div
               id="category-avengers-infinity-war"
-              class="card card-text-overlay card-vertical overflow-hidden shadow-light w-33"
+              class="card card-text-overlay card-vertical shadow-light"
             >
               <img
+                loading="lazy"
                 class="card-vertical-img"
                 src="assets/pngs/avenger-infinity.png"
                 alt="avengers infinity war poster png"
```

Comment:
Just a suggestion to not include poster png in the image alt, you can just include text that's part of image.

Reply:
Thanks, @anik31, for pointing this out. I will look into it.